DayrSuryan/DSEthiop15.xml

```diff
@@ -98,17 +100,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                               </item>
                            </list>
                         </collation>
-                        
-                        <condition key="good">Good condition, the leaves are slightly stained in many places.</condition>
-                        
+
+                        <condition key="good">Good condition,
+                          the leaves are slightly stained in many places. Parchment repair
+                        on <locus target="#4"/>.</condition>
```

Comment:
The information on the parchment repair (if it occurred during the process of parchmenting) should be included in a `<p>` in `<support>`

Reply:
Ok.